index.bs

```diff
@@ -11992,14 +11992,13 @@ Note: Future versions of the ECMAScript specification may define a total order f
 
     The name of each property that appears to exist due to an object supporting indexed properties
     is an <dfn id="dfn-array-index-property-name" export>array index property name</dfn>,
-    which is a property name |P| such that [=Type=](|P|) is String
-    and for which the following algorithm returns <emu-val>true</emu-val>:
+    which is a property name |P| for which the following algorithm returns true:
 
-    1.  Assert: [=Type=](|P|) is String.
+    1.  If [=Type=](|P|) is not String, then return false.
     1.  Let |i| be [=!=] [=ToUint32=](|P|).
     1.  Let |s| be [=!=] [=ToString=](|i|).
-    1.  If |s| ≠ |P| or |i| = 2<sup>32</sup> − 1, then return <emu-val>false</emu-val>.
-    1.  Return <emu-val>true</emu-val>.
+    1.  If |s| is not equal to |P| or |i| = 2<sup>32</sup> − 1, then return false.
```

Comment:
Figured keeping "=" sort of made sense given these were ints, but then they're not abstract math values but actual ES values, so maybe "is" or "is equal to" are more appropriate.